### ISSUE TITLE
Broken JDK compatibility

### ISSUE THREAD

**Issue by autoluminescent** [jackdoe/turtle#7]
The releases 2.0.7-BETA and above are no longer compatible with the JDK I'm using. This is the Java version:

```
java version "1.8.0_92"
Java(TM) SE Runtime Environment (build 1.8.0_92-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.92-b14, mixed mode)
```

And this is the error I get with 2.0.7-BETA:

```
java.lang.NoSuchMethodError: java.nio.CharBuffer.clear()Ljava/nio/CharBuffer;

	at bz.turtle.readable.input.Feature.setByteBuffer(Feature.java:236)
	at bz.turtle.readable.input.Feature.setName(Feature.java:197)
	at bz.turtle.readable.input.Feature.<init>(Feature.java:54)
	at bz.turtle.readable.input.Feature.<init>(Feature.java:45)
```

And with 2.0.9-BETA I get a different, but similar one:

```
java.lang.NoSuchMethodError: java.nio.CharBuffer.position(I)Ljava/nio/CharBuffer;

	at bz.turtle.readable.input.Feature.setByteBuffer(Feature.java:230)
	at bz.turtle.readable.input.Feature.setName(Feature.java:190)
	at bz.turtle.readable.input.Feature.<init>(Feature.java:47)
	at bz.turtle.readable.input.Feature.<init>(Feature.java:38)
```

It's worth noting that if I build the project myself on this JDK, it works correctly, so it seems to be some cross-version incompatibility.

**Comment by jackdoe** [comment]
ouch, released version 2.0.10-BETA that I compiled with jdk8, please tell me if it works for you

**Comment by autoluminescent** [comment]
Thanks @jackdoe! Still not working for me though. :\ I looked into the JAR's manifest, and it's still apparently built on JDK 11:

```
Manifest-Version: 1.0
Archiver-Version: Plexus Archiver
Created-By: Apache Maven 3.6.0
Built-By: jack
Build-Jdk: 11.0.3
```

**Comment by jackdoe** [comment]
haha sorry about that apparently JAVA_HOME and default java had a mismatch, i released 2.0.11-BETA that says 1.8 in the manifest

let me know if it works

**Comment by autoluminescent** [comment]
Yes, works perfectly. Thanks again!
